sources/org.osbuild.curl

```diff
@@ -139,31 +135,30 @@ def _quote_url(url: str) -> str:
 
 def gen_curl_download_config(config_path: pathlib.Path, chksum_desc_tuple: List[Tuple[str, Dict]], parallel=False):
     with open(config_path, "w", encoding="utf8") as fp:
-        # global options
         if parallel:
-            fp.write("parallel\n")
-        fp.write(textwrap.dedent(f"""\
+            fp.write("parallel\n\n")
```

Comment:
The arrangement of the fp.write and the string building is mildly confusing to me. Maybe move the parallel write down to just before the loop? That way you start with string building all in one spot, and carry on with all the fp.writes near each other.

Reply:
Thanks for the review, this was indeed a bit confusing -  I moved the code as suggested and tweaked the comments a bit. Hope that helps.